index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
I mean, the only way to implement smooth scrolling is by disabling the browsers scrolling and implementing your own. If something in the scrolling mechanism has a bug, then you'll end up with no scrolling. Not sure it's a risk I'd like to have.

It's also code that needs to be downloaded and parsed. The site currently takes more then 2 seconds to load, which is pretty long; not sure I'd like to patch more stuff onto that.

This also feels like something that _should_ be up to the browser; not the site. Separation of concern and all that.